osidb/models/flaw/label.py

```diff
@@ -76,19 +167,40 @@ class FlawCollaboratorState(models.TextChoices):
 
     contributor = models.CharField(max_length=255, blank=True)
 
+    relevant = models.BooleanField(default=True)
```

Comment:
What does it mean that a label is not relevant? Is such label needed?

Reply:
These was one of the changes that was not part of the initial spec.

In OSIM/OSIDB we allow affect deletion and also modifying the ps_components/ps_modules of an affect.
`product_family` labels (the ones created automatically) are marked as `irrelevant` if the affect that added the label was removed or the component/module is not longer valid.

I proposed to just change the state of the label to `SKIP` or even create a new one, but they want to keep the state field just in case it is needed even if the label is not longer relevant.

(the name can be changed, I'm quite bad at naming things 😆 )

Reply:
The reason they gave me is that `nothing should be erased`, and keeping the state field is just in case an analyst has already put some effort into it and wants to move it to `DONE` or `SKIP` when finished, instead of the label being removed or the sate changed. I can ask for a better justification 😆